### ISSUE TITLE
"context cancelled" errors when resuming paused k6 during VU ramp-up

### ISSUE THREAD

**Issue by na--** [loadimpact/k6#788]
When I `k6 pause` a script that's in a ramp-up stage and resume it a few seconds later, one or more `context cancelled` errors pop up in the terminal:
```
...
DEBU[0002] Local: Scaling...                             num=1
DEBU[0004] Local: Setting VUs                            vus=2
DEBU[0004] Local: Scaling...                             num=2
DEBU[0004] running thresholds                            m=check_failure_rate
DEBU[0004] running thresholds                            m=http_req_duration
DEBU[0005] Local: Setting VUs                            vus=3
DEBU[0005] Local: Scaling...                             num=3
DEBU[0006] Local: Setting paused                         paused=true
DEBU[0006] PATCH /v1/status                              status=200
DEBU[0006] Local: Pausing!                              
...
DEBU[0018] Local: Setting paused                         paused=false
DEBU[0018] PATCH /v1/status                              status=200
DEBU[0018] Local: No longer paused                      
DEBU[0018] Local: Setting VUs                            vus=1
DEBU[0018] Local: Scaling...                             num=1
DEBU[0018] Local: Setting VUs                            vus=3
DEBU[0018] Local: Scaling...                             num=3
ERRO[0018] context cancelled
	a .../github.com/loadimpact/k6/samples/thresholds_readme_example.js:1:1(0)
 
ERRO[0018] context cancelled
	at .../github.com/loadimpact/k6/samples/thresholds_readme_example.js:1:1(0)
 
DEBU[0018] Local: Setting VUs                            vus=4
DEBU[0018] Local: Scaling...                             num=4
DEBU[0020] Local: Setting VUs                            vus=5
...
```

**Comment by MStoykov** [comment]
@na-- can we close this ... because I don't think this is relevant anymore :)

**Comment by na--** [comment]
huh, yeah, this should have been closed way back, with k6 v0.27.0...
